GDAPI/GDAPI.Tests/Objects/GeometryDash/GuidelineCollectionTests.cs

```diff
@@ -23,7 +23,7 @@ public void Stringification()
                 new Guideline(7.25, GuidelineColor.Green),
                 new Guideline(8.112, GuidelineColor.Yellow),
             });
-            Assert.AreEqual("1.5~1~3.2~0.9~4.7~0.8~6~1~7.25~1~8.112~0.9", guidelines.ToString());
+            Assert.AreEqual("1.5~1~3.2~0.9~4.7~1.1~6~1~7.25~1~8.112~0.9", guidelines.ToString());
```

Comment:
There is no reason to change the tests, the 0.8 value is found in the official levels' guidelines, which is the original orange I guess.